reporting/api/src/main/java/org/jboss/windup/reporting/rules/CreateApplicationListReportRuleProvider.java

```diff
@@ -30,6 +24,12 @@
 import org.ocpsoft.rewrite.config.ConfigurationBuilder;
 import org.ocpsoft.rewrite.context.EvaluationContext;
 
+import javax.inject.Inject;
+import javax.json.*;
+import java.nio.file.Path;
+import java.util.*;
```

Comment:
It seems like maybe these changes are coming from when you hadn't got the right code formatter.

Reply:
done